store/store.go

```diff
@@ -253,6 +270,8 @@ func (c *MongoStoreClient) EnsureIndexes() error {
 		log.Fatal(dataStoreAPIPrefix, fmt.Sprintf("Unable to create indexes: %s", err))
 	}
 
+	mongoIndexesValid.Set(1)
```

Comment:
This will always be `1` too - if the indexes are not created successfully, the process will be terminated and metrics will not be scraped.